src/engines/transit.ts

```diff
@@ -210,6 +218,71 @@ export class TransitVaultClient extends AbstractVaultClient {
         });
     }
 
+    /**
```

Comment:
run `yarn format`

Reply:
Made the changes and squashed the commits. I guess we need a very simple CONTRIBUTING.md page: just a quick list of instructions of what to run, i.e. running "docker-compose up" before integration testing etc. I'd gladly help but wouldn't know where to start.